src/pages/reservations/Resvervations.js

```diff
@@ -0,0 +1,38 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+
+import { reservationsThunk } from '../../slices/reservationSlice/reservationSlice';
+import { setAlert } from '../../slices/appSlice/appSlice';
+import ReservationCard from './ReservationCard';
+
+export default function Reservations() {
+  const dispatch = useDispatch();
+  const token = useSelector((state) => state.auth.token);
+  const reservations = useSelector((state) => state.reservation.reservations);
+
+  useEffect(() => {
+    dispatch(reservationsThunk(token)).then((res) => {
+      if (res.error) {
+        dispatch(setAlert(res.payload));
+      }
+    });
+  }, [dispatch, token]);
+
+  return (
+    <div>
+      <h1 className="reservations_title">My Reservations</h1>
```

Comment:
It would be very great if you use the font for My Reservations with the font-secondary that we agree on for important fonts

Reply:
Hi @heintayzar-hm, can you give me the taillwin code